projects/dotcms-field-elements/src/components/dot-select/dot-select.e2e.ts

```diff
@@ -1,93 +1,276 @@
+                element.setProperty('hint', wrongValue);
+                await page.waitForChanges();
+
+                const hintElement = await getHint(page);
+                expect(hintElement.innerHTML).toBe('<undefined></undefined>');
+            });
         });
 
-        describe('Events', () => {
-            beforeEach(async () => {
-                spyStatusChangeEvent = await page.spyOnEvent('statusChange');
-                spyValueChangeEvent = await page.spyOnEvent('valueChange');
+        describe('options', () => {
+            it('should render options', async () => {
+                const value = 'a|1,b|2,c|3';
+                element.setProperty('options', value);
+                await page.waitForChanges();
+
+                const optionElements = await getOptions(page);
+                expect(optionElements.length).toBe(3);
             });
 
-            it('should emit "statusChange" & "valueChange"', async () => {
+            it('should not render options', async () => {
+                const optionElements = await getOptions(page);
+                expect(optionElements).toEqual([]);
+            });
+
+            it('should not break with invalid data', async () => {
+                const wrongValue = [{ a: 1 }];
+                element.setProperty('options', wrongValue);
+                await page.waitForChanges();
+
+                const optionElements = await getOptions(page);
+                expect(optionElements).toEqual([]);
+            });
+        });
+
+        describe('required & requiredMessage', () => {
+            it('should render required attribute in label', async () => {
+                element.setProperty('required', true);
+                await page.waitForChanges();
+                const labelElement = await getDotLabel(page);
+                expect(await labelElement.getProperty('required')).toBe(true);
+            });
+
+            it('should render required error msg', async () => {
+                element.setProperty('required', true);
+                element.setProperty('requiredMessage', 'test');
+                await page.waitForChanges();
+                const errorElement = await getError(page);
+                expect(errorElement.innerHTML).toBe('test');
+            });
+
+            it('should not render required error msg', async () => {
+                const errorElement = await getError(page);
+                expect(errorElement).toBe(null);
+            });
+
+            it('should not break with invalid data', async () => {
+                const wrongValue = [{ a: 1 }];
+                element.setProperty('required', wrongValue);
+                element.setProperty('requiredMessage', wrongValue);
+
+                await page.waitForChanges();
+
+                const errorElement = await getError(page);
+                expect(errorElement).toBeTruthy();
+                expect(errorElement.innerHTML).toBe('<undefined></undefined>');
```

Comment:
Why `<undefined>`? also we have a method to check html.

Reply:
when set a value different than `string` the inner html of 
https://github.com/dotCMS/core-web/blob/4eb42cf5381a770c729ac8420b55c08098fb9836/projects/dotcms-field-elements/src/utils/utils.tsx#L135

returns `<undefined></undefined>`

Reply:
But is returning `null`... how do we make it to return nothing?